components/remote-environment-broker/README.md

```diff
@@ -39,6 +39,22 @@ After normalization, the **displayName** contains only lowercase characters, num
 
 Before each commit, use the `before-commit.sh` script, which tests your changes.
 
+### Use environment variables
+Remote Environment Broker can work in 2 modes: as a cluster-scoped or namespace-scoped broker (see **APP_CLUSTER_SCOPED_BROKER_ENABLED**).
+In the first case, on creation of a Remote Environment, proper Cluster Service Classes will be created automatically and will be available for every namespace.
+In the second case, on creation of a Environment Mapping, namespaced-scoped broker is registered in the specific namespace and  services from this Remote Environment are visible in the namespace.
```

Comment:
I think that you should move this section to `Overview` or to a separate section `Temporary solution`. 
`Use environment variables` section should contain only information about `environment variables` without REB specifications.

Reply:
`services from this Remote Environment are visible in the namespace.`
shouldn't be?
`services from the Remote Environment are visible in this namespace.`
or?
`services from the Remote Environment are visible in the broker's namespace.`